workflows/pylint.yml

```diff
@@ -0,0 +1,31 @@
+name: Pylint
```

Comment:
I'd leave this in the `.github/workflows` directory rather than pulling it up, so that github-specific actions can be more easily `.gitignore`d.

Reply:
its almost mid-night my time so i will finish that up tomorrow but are you saying to add the .github/workflows to the .gitignore in my fork?

Reply:
No, I was saying that it appears you added a new file `workflows/pylint.yml` that copies the existing `pylint.yml` from `.github/workflows/` to a new top-level `workflows` directory, which isn't necessary, so you could get rid of the top-level `workflows` directory and its contents. 

No rush on the changes, this feature has no target date.